### ISSUE TITLE
yellow color seems out of place in Gotham color pallet

### ISSUE THREAD

**Issue by alexanderjeurissen** [whatyouhide/vim-gotham#23]
This especially shows in the Airline theme when going in insert mode.
Compared with the color hue's used in other parts of the UI and compared to the colors for normal and visual mode (bright yellow, and green respectively) the yellow color just seems off, lowering the saturation of the color helps but doesn't solve it completely.

below two screenshots that illustrate this, side note: I use the Gotham ergo theme for Iterm.

**the insert mode yellow:**
<img width="1280" alt="screen shot 2016-01-30 at 18 49 44" src="https://cloud.githubusercontent.com/assets/1220084/12697462/5168cc96-c784-11e5-89d2-1f2b302c347c.png">

**The profile overview of the Gotham ergo theme, note that the yellow color is the same in all iterm colors chemes:**
<img width="605" alt="screen shot 2016-01-30 at 19 02 52" src="https://cloud.githubusercontent.com/assets/1220084/12697465/5f2bd986-c784-11e5-96ee-f93c4c708c8e.png">

I'm curious as to what your opinion is on the mather and how this yellow color came about.

**Comment by whatyouhide** [comment]
@alexanderjeurissen I have absolutely no design knowledge of any kind so I basically just put together a bunch of colors that made sense to me together :). You may be right, the yellow could indeed look a little off. If you have any suggestions, I'm very open to them!

**Comment by alexanderjeurissen** [comment]
maybe using the bright red color in the airline theme instead of yellow. I'm not sure how that would look though so I'll have to try that out.

Is the yellow color used anywhere else besides the airline theme insert mode ? otherwise we could replace it with another color. I'll also have a go at experimenting with the saturation of the yellow color maybe I can tweak it into awesomeness.

**Comment by whatyouhide** [comment]
I haven't worked hands on on gotham for a while now, so I'm not sure if yellow is used only in the airline theme. Not sure about red in the airline theme, as red looks error-y and I'm not sure that would be good.

Maybe toying with the saturation of yellow is a safe approach if you manage to make it "keep the spirit" of gotham, whatever that means :D

**Comment by alexanderjeurissen** [comment]
@brucewayne what is your take on this ?

**Comment by alexanderjeurissen** [comment]
lol that username actually exists XD

**Comment by whatyouhide** [comment]
:smile:

**Comment by alexanderjeurissen** [comment]
I did some more research but changing the yellow hue / brightness only makes it weird compared to the green en orange colors which are also more "pastel" like.

I've also noticed the the yellow color is used in other places tmux uses yellow by default in the menus for example.I think that the best solution is changing the airline theme colors and not solve this in the terminal theme.

I also found another use for the yellow color as can be seen in my new tmux status line which fits in quite nicely with the rest of Gotham :smile: 
![screen shot 2016-02-06 at 01 26 46](https://cloud.githubusercontent.com/assets/1220084/12863002/e3e83bce-cc70-11e5-84cb-0742f7db1551.png)

long story short, I'll have a look at the airline theme colors and will update this issue if I find a solution or give up ;)

**Comment by whatyouhide** [comment]
Sounds like a plan :+1:

**Comment by jayniz** [comment]
> I also found another use for the yellow color as can be seen in my new tmux status line which fits in quite nicely with the rest of Gotham

@alexanderjeurissen wanted to ask what that neat, tmux status line is. then thought the guy might have his dotfiles on github. and indeed, he does. 👌thank you!

**Comment by whatyouhide** [comment]
Closing this in favor of #26.
